packages/deployer/subtasks/generate-imc.js

```diff
@@ -0,0 +1,83 @@
+const fs = require('fs');
```

Comment:
Can we rename this file to something more intuitive? First time you see "imc" its hard to tell what it is.
I'd rather something verbose, but self explanatory.